shadows/framework/src/main/java/org/robolectric/shadows/ShadowFileProvider.java

```diff
@@ -0,0 +1,35 @@
+package org.robolectric.shadows;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Build;
+import androidx.annotation.NonNull;
+import java.io.File;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+/**
+ * A shadow implementation of androidx.core.content.FileProvider.
+ *
+ * <p>Currently only supports getUriForFile(). The returned Uri will always have "content" as its
+ * scheme, authority matching to that of the given argument, and path matching to the absolute path
+ * of the given file instance.
+ */
+@Implements(
+    className = "androidx.core.content.FileProvider",
+    isInAndroidSdk = false,
+    minSdk = Build.VERSION_CODES.LOLLIPOP)
+@SuppressWarnings("robolectric.internal.IgnoreMissingClass")
+public class ShadowFileProvider extends ShadowContentProvider {
```

Comment:
I've [verified this on my project](https://github.com/MrHadiSatrio/Journal3/commit/d3adfef091e5913003a6b52e3441190f5c6a6ecf) through a `SNAPSHOT` build.